src/transports/ipc.rs

```diff
@@ -255,7 +255,7 @@ impl Future for WriteStream {
           }
         },
         WriteState::Writing { ref buffer, ref mut current_pos } => {
-          try_ready!(Ok(self.write.poll_write()));
+          try_ready!(Ok(self.write.poll_write()) as Result<_, ()>);
```

Comment:
after importing `tokio_timer` I had to explicitly specify Result error. Don't know why...